css/style.css

```diff
@@ -345,3 +345,8 @@ body.content ul ul{
   height: auto;
   max-width: 100%;
 }
+.Chat_buttons{
+  display: flex;
+  margin: 0 3px 0 3px;
+  justify-content: center;
+}

```

Comment:
this is not needed except the margin, please simply use `.chat.buttons` and add the additional `chat` class to the `p` element.

Reply:
I will update the changes requested.